woocommerce/woocommerce
# [Accessibility] Fix address heading level on My Account page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43630 .
Closes #43633 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Before testing this PR, please make sure the logged user has both Billing and Shipping addresses registered.
2. Go to My Account -> Addresses page.
3. Verify both the "Billing address" and "Shipping address" headings are `<h2>`.
4. Click on the "Edit" link to edit the billing address.
5. Verify the "Billing address" heading on the edit page is `<h2>`.
6. Click on the "Edit" link to edit the shipping address.
7.  Verify the "Shipping address" heading on the edit page is `<h2>`.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
